specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2018-05-05-preview/AlertsManagement.json

```diff
@@ -164,7 +164,7 @@
         },
         "x-ms-examples": {
           "GetById": {
-            "$ref": "./examples/Alerts_GetbyId.json"
+            "$ref": "./examples/Alerts_GetById.json"
```

Comment:
This file name is not changed.

Reply:
Actually do not change the file names. It creates unnecessary hassle to make sure that the file exists.

Reply:
Spell check fixes in description and readme are good. Let's not rename the files. What we have is good enough.
Please address this feedback to get the PR merged.

Reply:
I know it seems silly, but it's one of the things that gets flagged in #4836. If it's not fixed here, then it's going to need to be excluded later. There have already been a few PRs to fix these little casing things landed

Reply:
I did find that the modelValidation.js crashes hard when I did rename the reference without the file 😝